stake-pool/py/tests/test_a_time_sensitive.py

```diff
@@ -51,21 +68,31 @@ async def test_increase_decrease_this_is_very_slow(async_client, validators, pay
     for validator in validator_list.validators:
         assert validator.last_update_epoch != 0
         assert validator.transient_stake_lamports == 0
-        assert validator.active_stake_lamports == increase_amount + minimum_amount
+        assert validator.active_stake_lamports == increase_amount + minimum_amount + stake_rent_exemption
+
+    def decrease(validator, decrease_amount, ephemeral_stake_seed=None):
+        return decrease_validator_stake(async_client, payer, payer, stake_pool_address, validator,
```

Comment:
nit: I'm not sure I understand the need for this function -- can you remove it and just use `decrease_validator_stake` directly?